tekton/pipeline_run_test.go

```diff
@@ -247,4 +238,35 @@ var _ = Describe("PipelineRun", func() {
 			Expect(releasePipelineRun.Spec.Params).Should(ContainElement(HaveField("Value.StringVal", Equal(string(jsonSpec)))))
 		})
 	})
+
+	Context("WithReleaseStrategy handles all limbs of PVC conditional branch", func() {
```

Comment:
These look good, but I am not sure how they relate. Is this just you improving test coverage in the file for other functions you didn't touch?

Reply:
Yeah, it's technically unrelated to hacbs-1193.
However, I was making edits to `pipeline_run.go` and noticed missing coverage that was super trivial.
So I wanted to leave the file better than I found it.
We could remove these lines if you want, I don't care.

Reply:
No, I think they are great to have, just wanted to double check. In an ideal world I think they are part of a separate commit, but could still be in the same PR (one commit with all the PaC stuff, one commit fixing this test coverage) but making it two separate commits may be a pain now and I don't think it matters too much